build/package/windows/nri-amd64-installer/Product.wxs

```diff
@@ -0,0 +1,99 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
+    <?define Version = "$(var.IntegrationVersion)" ?>
+    <?define ProductId = "*" ?>
+    <?define UpgradeCode = "{f2334d3a-10f1-4a4c-9185-08d197f95b2b}" ?>

```

Comment:
The upradedCode should be unique , i checked that the nri-redis already have this one. This should be updated before merge